.jenkins

```diff
@@ -384,7 +384,7 @@ pipeline {
                                 -DCMAKE_CXX_STANDARD=17 \
                                 -DKokkos_ARCH_NATIVE=ON \
                                 -DKokkos_ENABLE_COMPILER_WARNINGS=ON \
-                                -DKokkos_ENABLE_OPENMP=OFF \
+                                -DKokkos_ENABLE_OPENMP=ON \
```

Comment:
I would rather not mess with that build in that PR.
I expect a CPU-only build should be able to expose the issue you are fixing in this PR.